### PR TITLE
Add the fit to the McConnell&Ma (2013) Mstar-MBH relation

### PR DESCRIPTION
**Pull request by MatthieuSchaller**
Does what it says on the tin.

### CODE REVIEW

File: data/GalaxyStellarMassBlackHoleMass/conversion/convertMcConnell2013.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+# Create the x-data
+M_star = np.logspace(np.log10(M_min), np.log10(M_max), 50) * unyt.Solar_Mass  # Msun
+
+# Create the y-data (fit from the paper)
+M_BH = 8.46 + 1.05 * np.log10(M_star / (1e11 * unyt.Solar_Mass))
+M_BH = (10.0 ** M_BH) * unyt.Solar_Mass  # Msun
+
+# Meta-data
+comment = f"Data obtained assuming the total stellar mass is the same as the bulge mass. No cosmology correction needed."
```
Comment (JBorrow) [inline data/GalaxyStellarMassBlackHoleMass/conversion/convertMcConnell2013.py]: Please multi-line and remove `f`:

```
comment = (
    "Data obtained assuming the total stellar mass is the same as "
    "the bulge mass. No cosmology correction needed."
)

**Comment by JBorrow** [comment]
Thanks!

**Comment by JBorrow** [comment]
What comparison should we use for bulge mass? Still M_* 30kpc?

**Comment by MatthieuSchaller** [comment]
Yes. What they mean by bulge here is the non-rotating part of a galaxy. So if you have an elliptical, that's the whole galaxy. And at high masses you ought to have mostly ellipiticals.

**Pull request merged**